metricbeat/module/coredns/_meta/kibana/7/dashboard/Metricbeat-coredns-overview.json

```diff
@@ -0,0 +1,1395 @@
+                "mode": "stacked",
+                "show": true,
+                "showCircles": true,
+                "type": "histogram",
+                "valueAxis": "ValueAxis-1"
+              },
+              {
+                "data": {
+                  "id": "6",
+                  "label": "Average coredns.stats.dns.request.duration.ns.bucket.4096000000"
+                },
+                "drawLinesBetweenPoints": true,
+                "mode": "stacked",
+                "show": true,
+                "showCircles": true,
+                "type": "histogram",
+                "valueAxis": "ValueAxis-1"
+              },
+              {
+                "data": {
+                  "id": "5",
+                  "label": "Average coredns.stats.dns.request.duration.ns.bucket.2048000000"
+                },
+                "drawLinesBetweenPoints": true,
+                "mode": "stacked",
+                "show": true,
+                "showCircles": true,
+                "type": "histogram",
+                "valueAxis": "ValueAxis-1"
+              },
+              {
+                "data": {
+                  "id": "4",
+                  "label": "Average coredns.stats.dns.request.duration.ns.bucket.1024000000"
+                },
+                "drawLinesBetweenPoints": true,
+                "mode": "stacked",
+                "show": true,
+                "showCircles": true,
+                "type": "histogram",
+                "valueAxis": "ValueAxis-1"
+              }
+            ],
+            "times": [],
+            "type": "line",
+            "valueAxes": [
+              {
+                "id": "ValueAxis-1",
+                "labels": {
+                  "filter": false,
+                  "rotate": 0,
+                  "show": true,
+                  "truncate": 100
+                },
+                "name": "LeftAxis-1",
+                "position": "left",
+                "scale": {
+                  "mode": "normal",
+                  "type": "linear"
+                },
+                "show": true,
+                "style": {},
+                "title": {
+                  "text": "Count"
+                },
+                "type": "value"
+              }
+            ]
+          },
+          "title": "Request Duration [Metricbeat CoreDNS] ECS",
+          "type": "line"
+        }
+      },
+      "id": "e3e73b00-53ec-11e9-b466-9be470bbd327-ecs",
+      "type": "visualization",
+      "updated_at": "2019-04-25T22:47:01.782Z",
+      "version": "WzI2LDFd"
+    },
+    {
+      "attributes": {
+        "description": "",
+        "kibanaSavedObjectMeta": {
+          "searchSourceJSON": {
+            "filter": [],
+            "index": "metricbeat-*",
+            "query": {
+              "language": "lucene",
+              "query": ""
+            }
+          }
+        },
+        "title": "Request Size Bytes [Metricbeat CoreDNS] ECS",
```

Comment:
pretty much like the request duration case.
This time metric is provided by `proto`, `server`, and `zone`
Averaging different values can be misleading